setup.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+setup(
+    name='quickcache',
+    version='0.0.1',
```

Comment:
`0.0.1` gives the impression is this is super new and not at all tested. i feel like it's a little more battle ready than this implies